doc/ref/configuration/master.rst

```diff
@@ -2353,6 +2324,21 @@ strategy between different sources. It accepts 4 values:
 
   Guesses the best strategy based on the "renderer" setting.
 
+``pillar_merge_lists``
+----------------------------------
+
+.. versionadded:: 2015.8.0
+
+Default: ``True``
```

Comment:
Correct. @seanjnkns would you consider opening a quick PR to fix that mistake in the docs? 2015.8 would be a good target, since this was backported to 2015.8.